test/remote/gateways/remote_wepay_test.rb

```diff
@@ -97,11 +99,36 @@ def test_failed_authorize
     assert_failure response
   end
 
-  def test_successful_store
+  def test_successful_store_via_create_with_cvv
+    # POST /credit_card/create
     response = @gateway.store(@credit_card, @options)
     assert_success response
   end
 
+  def test_successful_store_via_transfer_without_cvv
+    # special permission required
+    # POST /credit_card/transfer
+    response = @gateway.store(@credit_card_without_cvv, @options.merge(recurring: true))
+    assert_success response
+  end
+
+  def test_unsuccessful_store_via_create
+    # without cvv
+    response = @gateway.store(@credit_card_without_cvv, @options)
```

Comment:
Nitpick: Could name the test `test_unsuccessful_store_via_create_with_cvv` and nix the `# without cvv` comment.

Reply:
Sure -- done.